src/main/java/com/mraof/minestuck/effects/CreativeShockEffect.java

```diff
@@ -62,6 +66,12 @@ public static void stopElytraFlying(Player player, int survivalAmplifierThreshol
 			player.stopFallFlying();
 	}
 	
+	@Override
+	public List<ItemStack> getCurativeItems()
+	{
+		return new ArrayList<>(); //prevent milk from curing creative shock
```

Comment:
Changed!

Reply:
Now that I've got my dev environment in front of me, I took a closer look at the function you override and realized a problem with my suggestion. Returning an immutable list from this function will cause `IForgeMobEffectInstance.addCurativeItem()` to crash for that effect. It would then be better to revert the change.

This still seems to be the case in latest neoforge, so it could also be relevant to bring it up or suggest a fix in their end.